schemas/components/cs3/difficulty.ksy

```diff
@@ -0,0 +1,14 @@
+meta:
+  id: difficulty
+  endian: le
+seq:
+  - id: difficulty_id
+    type: u2
+    enum: difficulties
+enums:
+  difficulties:
```

Comment:
Should this be in common or just cs3? I honestly don't remember the difficulty id for cs1-2 and not sure about haji

Reply:
CS1/2 don't have a Very Easy difficulty. CS4 has the same ones as CS3 so I assume the values would be shared too (though I have no way to verify). It's probably easiest to keep it separate.